### PR TITLE
tweak mold indicator wording

### PR DESCRIPTION
**Pull request by KTibow**
## Proposed change

I found the mold indicator documentation a bit unclear. The first paragraph is really long at the font size it shows as, and doesn't make it clear what type of sensor it is. This PR updates it to make it more clear what it calculates and how it does it.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Revised descriptions for the Mold Indicator sensor integration to highlight its predictive capabilities regarding humidity.
	- Added a "Background" section to explain the sensor's temperature prediction and humidity calculation methods.
	- Streamlined information for clarity and conciseness, enhancing user understanding of the sensor's functionality.

### CODE REVIEW

File: source/_integrations/mold_indicator.markdown [hunk 1]
```diff
@@ -59,9 +59,16 @@ calibration_factor:
 
 In this case, the weather forecast temperature sensor is used for the outside temperature.
 
+## Background
+
+This sensor first predicts the temperature near the given surface by estimating it from the indoor and outdoor temperature. The predicted temperature starts with the indoor temperature, then either brings it closer to the outdoor temperature (if the calibration factor is positive, indicating the surface is influenced by the outdoors) or away (if the calibration factor is negative, indicating the surface is inversely correlated to the outdoors temperature).
+
+From there, the sensor calculates the relative humidity near the given surface based off of the dew point.
```
[the remark this answers is not in the document]
Comment (KTibow) [reply source/_integrations/mold_indicator.markdown]: can you reply to these comments?

File: source/_integrations/mold_indicator.markdown [hunk 2]
```diff
@@ -59,9 +59,16 @@ calibration_factor:
...
+
 ## Calibration
 
 The Mold Indicator sensor integration needs to be calibrated in order to estimate the temperature at the critical point from the outdoor and indoor temperature. First find the coldest surface in the room (critical point), which is typically near the window frames, but depends on the insulation of your home. For calibration you need to measure the temperature at this critical point and simultaneously write down the values for the indoor- and outdoor temperature sensors used for the Mold Indicator. Be sure that there is a significant indoor to outdoor temperature difference to get the best calibration results.
```
[the remark this answers is not in the document]
Comment (KTibow) [reply source/_integrations/mold_indicator.markdown:70]: out of scope

File: source/_integrations/mold_indicator.markdown [hunk 3]
```diff
@@ -15,9 +15,9 @@ related:
...
+The Mold Indicator sensor {% term integration %} uses data from two temperature sensors and a humidity sensor to predict the humidity near a cold surface. High humidity may lead to mold growth at more than ~70% and causes condensation at 100%.
 
-The sensor data may be used e.g., to signal bad air quality (too high air humidity) or to automate operation of indoor air humidifiers to keep the indoor humidity at an optimum.
+The sensor data may be used to e.g. detect bad air quality (high humidity) or automate operation of indoor air humidifiers to keep the indoor humidity at an optimum.
```
Comment (c0ffeeca7) [inline source/_integrations/mold_indicator.markdown]:
```suggestion
The sensor data may be used, for example, to detect bad air quality (high humidity) or automate the operation of indoor air humidifiers to keep the indoor humidity at an optimum.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @KTibow :+1:

**Pull request merged**